cdap-docs/_common/common-build.sh

```diff
@@ -236,12 +225,6 @@ function make_zip() {
   zip -qr ${ZIP_DIR_NAME}.zip ${PROJECT_VERSION}/* --exclude .DS_Store
```

Comment:
Why are we using a license-encumbered format for this artifact? This is an additional dependency that not all systems will have, just for building docs. It just *happens* that our build hosts all have a `zip` binary. If we're going to keep this, as is, then we need to add a capability to our build agents on `zip` and have the builds depend on agents with this capability. Hidden, undocumented dependencies are the devil.